lib/canvas_api/rb_graphql_helpers.rb

```diff
@@ -161,11 +174,11 @@ def is_basic_type(type)
     end
 
     def no_brackets(str)
```

Comment:
Since this method now removes periods as well, `no_brackets` doesn't seem to be an accurate name.

Reply:
Changed name